generators/client-2/templates/src/main/webapp/app/account/activate/_activate.component.ts

```diff
@@ -23,4 +29,14 @@ export class ActivateComponent implements OnInit {
         });
     }
 
+    login(template) {
+        console.log('login activate');
+        this.modalRef = this.modalService.open(template);
```

Comment:
we need a better way for this. lets just use the loginService like before and do the result handling within that and return the modelRef from there